### ISSUE TITLE
Problem with the Music of the main menu and background music during the game

### ISSUE THREAD

**Issue by 32Dexter** [notaz/pcsx_rearmed#111]
Hello everyone,
I've been having problems with the pcsx-rearmed emulator in Retropie on Raspberry Pi 3 B and the game Fisherman's Bait 3. I detect a problem with the music in the main menu, you do not hear anything at all, but the sound effects of the menu move work .
While when you enter the game you should hear background music but instead you hear nothing, you only hear the sound effects of the game.
Can you fix it? Thank you

**Comment by 32Dexter** [comment]
I have found that the problem is due to the BIOS send.

**Comment by fraf** [comment]
In the CIA 3DS version (1.7.5), I don't have any music playing from CD Track in game (wipeout, tekken, moto racer, etc.)

**Comment by gingerbeardman** [comment]
> I have found that the problem is due to the BIOS send.

@32Dexter can you please expand on your solution? DO you mean that you use a different BIOS and the problem goes away? If so, which BIOS did you use to get things to work? Thanks

**Comment by fraf** [comment]
I don't think so. But the whole ps1 bios list is here : http://www.planetemu.net/roms/sony-playstation-bios?screenshots=true&page=0

**Comment by notaz** [comment]
The cdda code has seen many updates so I expect this to work now. Feel free to reopen if there is still an issue.
